includes/class-zoninator-api-controller.php

```diff
@@ -417,7 +417,7 @@ private function _get_param( $object, $var, $default = '', $sanitize_callback =
 	 *
 	 * @return array[]
 	 */
-	public function _params_for_create_zone() {
+	public function params_for_create_zone() {
```

Comment:
This, and many other changes in this file, are renaming public methods, which means that it would be a breaking change.

Are there other changes around that would make sense to do in a major release, or should these method renames be held back (and allow the files with the underscore-prefixed methods to be ignored from the PHPCS), until a later time?

Reply:
For each renamed method, I checked where they were being used and updated all the references. The only potential issues with these are if they are used outside of the plugin context, by a third-party developer. Most of these methods are public because either they had the default visibility, or because [they are used as a callback](https://github.com/Automattic/zoninator/pull/84/files/a75b04570ba816558daffa540a30d5a0511b9884..8e4048255965b666116c9ea8ac2987addb7ea465#diff-f9354c01816a57ed9450081a8fa6c097b48c744721a7138b71b4b8117062abd2R67) and might need to have `public` visibility for the method call to work. 

Regarding other changes that we can include for a major release, I don't have any. I really don't think these methods are used outside Zoninator, and I think that ideally, they should even have private visibility. However, I haven't test changing the visibility of these specific methods to private. 

I can definitely revert the renaming of the public methods if you think it is the best and safer approach.

Reply:
Since it is technically a breaking change, however unlikely it is that someone might be calling them in their code, we should follow SemVer (that's why we have it), and do a major bump accordingly.